clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -548,6 +552,79 @@ public byte[] get(ByteBuffer keyBuffer) {
     }
   }
 
+  public List<byte[]> multiGet(List<byte[]> keys) {
+    readCloseRWLock.readLock().lock();
+    try {
+      makeSureRocksDBIsStillOpen();
+      return rocksDB.multiGetAsList(keys);
+    } catch (RocksDBException e) {
+      throw new VeniceException("Failed to get value from store: " + storeName + ", partition id: " + partitionId, e);
+    } finally {
+      readCloseRWLock.readLock().unlock();
+    }
+  }
+
+  public List<ByteBuffer> multiGet(List<ByteBuffer> keys, List<ByteBuffer> values) {
+    readCloseRWLock.readLock().lock();
+    try {
+      makeSureRocksDBIsStillOpen();
+      List<ByteBufferGetStatus> statusList = rocksDB.multiGetByteBuffers(keys, values);
```

Comment:
This can ` * @throws IllegalArgumentException thrown if the number of passed keys, passed values and
   * passed column family handles do not match.` Can we add some assert before this call or catch the exception ?

Reply:
RocksDB internally will do all the checks:
https://github.com/facebook/rocksdb/blob/main/java/src/main/java/org/rocksdb/RocksDB.java#L2347
That is the reason I don't want to perform the same check again here.